dgbpy/torch_classes.py

```diff
@@ -46,12 +46,16 @@ def hasFastprogress():
 class OnnxModel():
     def __init__(self, filepath : str):
         self.name = filepath
+        providers = [dgbkeys.onnxcudastr, dgbkeys.onnxcpustr]
+        try:
+            self.session = rt.InferenceSession(self.name, providers=providers)
+        except RuntimeError:
+            self.session = rt.InferenceSession(self.name, providers=[dgbkeys.onnxcpustr])
```

Comment:
Some onnx package versions do not automatically change to the cpu provider when the cuda provider is not available.
The try-except block handles this case